Assets/Scripts/GlobalValues/Alphabet.cs

```diff
@@ -4,7 +4,7 @@
 
 public class Alphabet
 {
-    public static char[] alphabet = "TEST".ToCharArray();
+    public static char[] alphabet = "ABCDEFGHIJKLMNOPQRSTVWXYZ".ToCharArray();
```

Comment:
we both made this change - i will solve the conflict after merging yours

Reply:
might be ok if its exactly the same